suite-native/module-dev-utils/src/screens/DemoScreen.tsx

```diff
@@ -70,14 +61,6 @@ export const DemoScreen = () => {
                 <VStack>
                     <Text variant="titleSmall">Transaction Notifications:</Text>
                     <Text variant="titleSmall">Toast Notifications:</Text>
-                    <VStack
```

Comment:
We won't have notifications in component demo? 😢

Reply:
No, since the data is read directly from the redux state. I could rework it a bit, but it would need some extra work. Do we really need it?

Reply:
Not really priority now